python_modules/libraries/dagster-looker/dagster_looker/dagster_looker_translator.py

```diff
@@ -6,11 +6,19 @@
 
 from dagster import AssetKey
 from dagster._annotations import experimental, public
+from dagster._core.definitions.utils import (
+    is_valid_definition_tag_key,
+    is_valid_owner,
+    is_valid_team_owner,
+)
 from sqlglot import exp, parse_one, to_table
 from sqlglot.optimizer import Scope, build_scope, optimize
 
 LookMLStructureType = Literal["dashboard", "explore", "table", "view"]
 
+DAGSTER_LOOKER_OWNER_TAG_PREFIX = "owner:"
+DAGSTER_LOOKER_GROUP_TAG_PREFIX = "group:"
```

Comment:
group is a very Dagster-specific concept: I wonder if we should call this "dagster_group"? not sure on this